README.md

```diff
@@ -43,6 +43,12 @@ Login credentials: `admin@example.com`/`admin`
 * Follow us on [Twitter](https://twitter.com/getsaleor?lang=en)
 * Check our latest blog posts on [Medium](https://medium.com/saleor)
 
+## 🗺 Translate
+
+Saleor is available in 30 languages translated by our community, 
+the repository gets synchronized weekly to the latest contributions on 
```

Comment:
I'm also missing some invitation to contribute in the copy. 
Maybe something like
"
Did you know that Saleor is available in almost 30 languages, translated entirely by our community? 
If you'd like to help us, you can join one of our translation teams on [the localization platform Transifex](https://www.transifex.com/mirumee/saleor-1/languages/).
The repository gets synchronized weekly with the latest contributions.
"